### DISCUSSION TITLE
Timeline for 2.0?

### DISCUSSION THREAD

**Discussion by ProStashio** [RafaelGB/obsidian-db-folder#205] [General]
No rush, just curious when 2.0 is looking to be ready? I am looking forward to the ability of having pages / faster and more stable database folders with hundreds of notes!

**Comment by RafaelGB** [comment]
You can check the status in this kanban https://github.com/users/RafaelGB/projects/3/views/8

Estimated in 2 months, but will be beta versions while is ready with BRAT

The current roadmap issue is https://github.com/RafaelGB/obsidian-db-folder/issues/189

**Reply by ProStashio** [reply]
Ah perfect thanks for the info, sorry still getting used to how github works!
